src/Tester/Tester.csproj

```diff
@@ -61,6 +61,7 @@
     <Reference Include="FluentAssertions.Core">
       <HintPath>..\packages\FluentAssertions.4.0.0\lib\net45\FluentAssertions.Core.dll</HintPath>
     </Reference>
+    <Reference Include="FSharp.Core, Version=4.4.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, processorArchitecture=MSIL" />
```

Comment:
I've been using Visual Studio 2015 to build Orleans - I have just now tried building it  with VS 2013 update 4 and it failed due to an old version of the Nuget package manager.I am updating VS 2013 and will retry.
I think that the F# 4.0 project should build with VS 2013 if the F# 4.0 build tools are installed on the machine, despite the lack of tooling within the VS 2013 IDE.
I'll let you know how it turns out.